external-crates/move/crates/move-analyzer/trace-debug/src/extension.ts

```diff
@@ -23,9 +23,20 @@ const LOG_LEVEL = 'log';
  */
 const DEBUGGER_TYPE = 'move-debug';
 
-const MOVE_FILE_EXT = ".move";
-const BCODE_FILE_EXT = ".mvb";
+/**
+ * File extension for Move source files.
+ */
+const MOVE_FILE_EXT = '.move';
 
+/**
+ * The extension for JSON files.
+ */
+const JSON_FILE_EXT = ".json";
+
+/**
+ * Name of the trace file generated by the replay tool.
+ */
+const REPLAY_TRACE_FILE_NAME = 'trace';
```

Comment:
IMO the name of the generated file from a replay should include the transaction digest (unless everything is directory structured under the the transaction digest, e.g., `<tx_digest>/trace.json`, `<tx_digest>/sources` etc.

Reply:
This is the idea - everything in the directory "belongs" to a single transaction run, hence trace file name has no additional information embedded. It may not be a bad idea to qualify the whole directory with transaction digest. So, we would have the "main" directory (`./replay` or user-specified location) and inside we'd have subdirectories that correspond  transaction digests (and are named as such). Inside of these we'd have the `trace.json` file and all bytecode/sourcemaps (and optional source code) that's needed to trace-debug this transaction